app/views/komponenty/zalozky.njk

```diff
@@ -15,57 +15,231 @@
+                    <li>používatelia nebudú potrebovať zobraziť všetky sekcie naraz.</li>
+                </ul>
+                <p class="govuk-body">Záložky môžu dobre slúžiť pre používateľov, ktorí používajú dané informácie pravidelne, napríklad aktuálny covid automat.</p>
+
+                <h2 class="govuk-heading-l">Kedy nepoužívať tento komponent</h2>
+                <p class="govuk-body">Nepoužívajte záložky ako formu navigácie na stránke. Príliš veľa obsahu v záložkách môže spôsobiť pomalé načítavanie stránky, 
+                    v takomto prípade zvážte vytvorenie samostatných podstránok pre obsah alebo inú formu zobrazenia na stránke, napríklad cez štrukturované nadpisy.
+                </p>
+                <p class="govuk-body">Záložky skrývajú obsah pred používateľmi a nie každý si ich všimne alebo pochopí, ako fungujú.</p>
+                <p class="govuk-body">Nepoužívajte záložky, ak by používatelia mohli potrebovať:</p>
+                <ul class="govuk-list govuk-list--bullet">
+                    <li>prečítať si celý obsah, aby napríklad pochopili postup krok za krokom,</li>
+                    <li>porovnať informácie na rôznych kartách - nutnosť zapamätať si informácie a prepínať medzi nimi dozadu a dopredu môže byť frustrujúca a náročná.</li>
+                </ul>
+                <p class="govuk-body">Najprv otestujte obsah bez záložiek. Zvážte, či je lepšie:</p>
+                <ul class="govuk-list govuk-list--bullet">
+                    <li>zjednodušiť a zredukovať množstvo obsahu,</li>
+                    <li>rozdeliť obsah na viacero stránok,</li>
+                    <li>ponechať obsah na jednej stránke, oddelený nadpismi.</li>
+                </ul>
+
+                <h2 class="govuk-heading-l">Záložky, akordeóny a skrytý text</h2>
+                <p class="govuk-body">Záložky, akordeóny a skrytý text - tieto komponenty skrývajú časti obsahu a používateľ si ich môže zobraziť podľa potreby.</p>
+                <p class="govuk-body">Ak používatelia nepotrebujú naraz zobraziť viac ako jednu sekciu, zvážte použitie záložiek namiesto akordeónu.</p>
+                <p class="govuk-body">Mali by ste zohľadniť aj počet sekcií obsahu. Záložky sú usporiadané horizontálne (na desktope), 
+                    takže sa do nich nezmestí toľko sekcií ako do akordeónov, ktoré sa zobrazujú vertikálne.</p>
+                <p class="govuk-body">Karty môžu lepšie fungovať pre používateľov, ktorí potrebujú rýchlo prepínať medzi 2 sekciami.</p>
+                <p class="govuk-body">Ak existuje viacero súvisiacich častí obsahu, zvážte použitie záložiek namiesto komponentu skrytý obsah. 
+                    Komponent skrytý obsah môže byť lepší, ak máte len 1 alebo 2 časti obsahu, zároveň je menej vizuálne nápadný ako záložky, 
+                    takže má tendenciu fungovať lepšie pre obsah, ktorý nie je pre používateľov taký dôležitý.</p>
+
```

Comment:
opat tie divne zarovnania.. ked tak ten ukoncovaci tag a zaciatocny tag davaj na nove riadky ak to takto delis